frontend/Makefile

```diff
@@ -25,19 +25,17 @@ deploy:
 			-g ${RESOURCE_GROUP} \
 			-n "hcp-${USER}-dev-infra" \
```

Comment:
I think this needs to be updated, since the deployment name is the same as the resource group name now

Reply:
Thanks, added DEPLOYMENT name var just like other Makefiles and updated everything to call that instead